common/djangoapps/util/tests/test_help_context_processor.py

```diff
@@ -0,0 +1,69 @@
+"""
+Tests for help_context_processor.py
+"""
+
+import ConfigParser
+from mock import patch
+
+from django.conf import settings
+from django.test import TestCase
+
+from util.help_context_processor import common_doc_url
+
+
+CONFIG_FILE = open(settings.REPO_ROOT / "docs" / "lms_config.ini")
+CONFIG = ConfigParser.ConfigParser()
+CONFIG.readfp(CONFIG_FILE)
+
+
+class HelpContextProcessorTest(TestCase):
+    """
+    Tests for help_context_processor.py
+    """
+
+    def setUp(self):
+        super(HelpContextProcessorTest, self).setUp()
+
+    @staticmethod
+    def _get_doc_url(page_token=None):
+        """ Helper method for getting the doc url. """
+        return common_doc_url(None, CONFIG)['get_online_help_info'](page_token)['doc_url']
+
+    @staticmethod
+    def _get_pdf_url():
```

Comment:
Hmm.. I wonder if we can use this time to simply remove the `pdf_url` field.  I don't think it ever got used anywhere.